com.sap.cloud.lm.sl.common/src/main/java/com/sap/cloud/lm/sl/common/util/JsonUtil.java

```diff
@@ -56,23 +57,31 @@
         return fromJson(json, type, new ArrayList<T>(), Messages.CANNOT_CONVERT_JSON_STRING_TO_LIST);
     }
 
-    private static <T> T fromJson(String json, Type type, T defaultValue, String errorMessage) throws ParsingException {
+    private static <T> T fromJson(String json, Type type, T defaultValue, String errorMessage)
+        throws ParsingException {
         if (json == null || json.isEmpty()) {
             return defaultValue;
         }
         try {
-            return new Gson().fromJson(json, type);
+            Gson gson = new Gson();

```

Comment:
You don't need to create a new instance of Gson, because the create() method of GsonBuilder will create another one on the line below.